broker-daemon/broker-rpc/order-service/get-trade-history.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Retrieve information about completed trades
+ *
+ * @param {GrpcUnaryMethod~request} request - request object
+ * @param {Object} request.logger
+ * @param {BlockOrderWorker} request.blockOrderWorker
+ * @param {Object} responses
+ * @param {function} responses.GetTradeHistoryResponse - constructor for GetTradeHistoryResponse messages
+ * @return {responses.GetTradeHistoryResponse}
+ */
+
+async function getTradeHistory ({ logger, blockOrderWorker }, { GetTradeHistoryResponse }) {
+  try {
+    const { orders, fills } = await blockOrderWorker.getTrades()
+
+    const filteredOrders = orders.filter(order => order.status === 'COMPLETED' || order.status === 'EXECUTING')
```

Comment:
See my other comment, but I think this filtering would be better in the block order worker (and should use constants for status)